code/src/Components/App.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import data from "../data.json";
+import stretchGoal from "../stretch-goal.json";
+import { Album } from "./Album";
```

Comment:
You don't need the brackets when you are importing only one thing
```suggestion
import Album from "./Album";
```